### PR TITLE
PIP-936-bam-counts

### PR DESCRIPTION
**Pull request by ottojolanki**
bamcounts task + tests

### CODE REVIEW

File: tests/unit/wdl/test_bamcounts.wdl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+workflow test_bamcounts {
+    input {
+        File bam
+        Resources resources
+    }
+
+    call bamcounts.bamcounts {
+        input:
+            bam=bam,
+            resources=resources
```
Comment (keenangraham) [inline tests/unit/wdl/test_bamcounts.wdl]: Give me a comma!

Comment (keenangraham) [inline tests/unit/wdl/test_bamcounts.wdl]:
```suggestion
            resources=resources,
```

File: wdl/tasks/bamcounts.wdl [hunk 2]
```diff
@@ -0,0 +1,29 @@
+version 1.0
```
Comment (keenangraham) [inline wdl/tasks/bamcounts.wdl]: Put this in stampipes.

File: wdl/subworkflows/get_bamcounts.wdl [hunk 3]
```diff
@@ -0,0 +1,22 @@
+version 1.0
```
Comment (keenangraham) [inline wdl/subworkflows/get_bamcounts.wdl]: Got to delete this file now.

File: tests/integration/test_bamcounts.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+- name: test_get_bamcounts
```
Comment (keenangraham) [inline tests/integration/test_bamcounts.yml]: It looks like by convention we should add this to stampipes.yml as well. Can leave wdl name the same.

Comment (ottojolanki) [reply tests/integration/test_bamcounts.yml]: should be good now, sorry about that.

**Pull request merged**